crates/router/src/connector/aci/transformers.rs

```diff
@@ -47,16 +49,43 @@ pub struct AciCancelRequest {
     pub payment_type: AciPaymentType,
 }
 
-#[derive(Debug, Clone, Eq, PartialEq, Serialize)]
+#[derive(Debug, Clone, Serialize)]
 #[serde(untagged)]
 pub enum PaymentDetails {
     #[serde(rename = "card")]
-    Card(CardDetails),
+    AciCard(CardDetails),
+    Eps(BankRedirectionPMData),
+    Ideal(BankRedirectionPMData),
+    Giropay(BankRedirectionPMData),
+    Sofort(BankRedirectionPMData),
     #[serde(rename = "bank")]
     Wallet,
     Klarna,
-    #[serde(rename = "bankRedirect")]
-    BankRedirect,
+}
+
+#[derive(Debug, Clone, Serialize)]
```

Comment:
add `#[serde(rename_all = "camelCase")]`

For payment_brand, shopper_result_url you don't need separate serde rename. For others keep the serde rename as it is.